lib/delivery.server.js

```diff
@@ -146,7 +152,7 @@ Delivery.prototype.subscribe = function() {
 
   //server sending to client
   this.socket.on('send.success', function(uid){
-    _this.pubSub.publish('send.success', uid);
+    _this.pubSub.publish('receive.success', uid);
```

Comment:
I don't remember exactly, but when I send a file from a node client to a server,  the send event or the receive event (I don't remember which one) it is called twice. Then, looking at the events I found this issue and I though that if you receive from the socket a send.success event it means that the client has send the file, but you're receiving, so you have to trigger the receive.success event.
Maybe I'm wrong, but it worked for me